### PR TITLE
Fix of: "[NGRM] - Axes2D. Add missing properties to set background an…

### PR DESCRIPTION
**Pull request by nilscb**
…d text color #1436"

### CODE REVIEW

File: react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts [hunk 1]
```diff
@@ -58,6 +58,7 @@ export interface Axes2DLayerProps<D> extends ExtendedLayerProps<D> {
     labelFontSize?: number;
     fontFamily?: string;
     axisColor?: Color;
+    backGroundColor?: Color;
```
Comment (hkfb) [inline react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts:61]: Suggest `backgroundColor`

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts (hunk 1 above)
Comment (nilscb) [reply react/src/lib/components/SubsurfaceViewer/layers/axes2d/axes2DLayer.ts]: Fixed. And also made the text sharper and more readable.

**Comment by hkfb** [comment]
The text looks more blurry to me in the diff?

**Comment by nilscb** [comment]
The png look more blurry because it is smoothed woth a gaussian blur. But the text on the axes should be sharper.

**Review by hkfb — APPROVED** [review]

**Pull request merged**